### DISCUSSION TITLE
Diagramm im Webinterface

### DISCUSSION THREAD

**Discussion by Tobbz2k** [tbnobody/OpenDTU#1991] [Q&A]
Ist es möglich sich das Diagramm vom kleinen LED Display in der Weboberfläche anzeigen zu lassen?

**Comment by ghost** [comment]
Hi,

Möglich ist es.

Du bist nicht der Einzigste , der eine Diagrammansicht gerne hätte; das kommt ab und zu hier.
Die Frage ist eher, kommt es überhaupt mal, da mangels Speicher am ESP32 wohlmöglich andere Sachen priorisiert sind.
Wünsche gibt es viele:
- mehr Displays (OLED)
- mehr Sprachen
- mehr wie 10 Wechselrichter, mehr Typen von WR
- das Eine oder Andere zusätzlich/geändert für Hausautomation... 

Wenn du die Diskussionen, Pull-Anfragen und Probleme anschaust, findest du das.
Ein Blick in nachfolgende Links zeigt dir Möglichkeiten, wenn du es selber integrieren willst:
[RaBa64/OpenDTU-Database](https://github.com/RaBa64/OpenDTU-Database) -> [Database Brunch](https://github.com/RaBa64/OpenDTU-Database/tree/Database)
Disskussion [#1504](https://github.com/tbnobody/OpenDTU/discussions/1504#discussioncomment-8375856)

**Reply by Tobbz2k** [reply]
Das man aufgrund der Hardware da recht beschränkt ist, ist mir schon klar. Daher hatte ich mich eh schon über dieses kleine Diagramm im Display gewundert. Aber dafür müssen die Daten dann ja eh schon vorhanden sein, daher dachte ich, dass es eigentlich nur ein Thema der "Anzeige" ist
